src/arcam/fmj/__init__.py

```diff
@@ -322,6 +327,8 @@ class DecodeModeMCH(IntOrTypeEnum):
         DecodeMode2CH.MCH_STEREO: bytes([16, 69]),
     },
     (ApiModel.API860_SERIES, 2): {},
+    (ApiModel.APISA_SERIES, 1): {},
+    (ApiModel.APISA_SERIES, 2): {},
```

Comment:
the following are blank as 2CH and MCH decoding are unavailable for the SA models @elupus mentioned in #4